ci/check-documentation.py

```diff
@@ -114,6 +116,21 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
+        # check <omz_dir> reference validity
+
+        for link in doc_page.code_spans():
+            if link.startswith(omz_prefix):
+                file_relative_path = link[len(omz_prefix):]
+                file_path = OMZ_ROOT / file_relative_path
```

Comment:
I think it would make sense to rename these to `target_path_rel` and `target_path`, which would be consistent with the preceding loop, as well as more accurate (since these path do not necessarily point to a file).